R/xgboost-deployment.R

```diff
@@ -7,16 +7,15 @@
 #' \item Push these predictions to SQL Server or CSV
 #' }
 #' @docType class
-#' @usage XGBoostDeployment(type, df, grainCol, testWindowCol, 
-#' predictedCol, impute, debug)
+#' @usage XGBoostDeployment(type, df, grainCol,
+#' predictedCol, impute, debug, cores, modelName)
 #' @import caret
 #' @import doParallel
 #' @import xgboost
 #' @importFrom R6 R6Class
 #' @param type The type of model (must be multiclass)
 #' @param df Dataframe whose columns are used for new predictions
 #' @param grainCol The dataframe's column that has IDs pertaining to the grain
-#' @param testWindowCol (Depreciated) Predictions will be made for all rows.
 #' @param predictedCol Column that you want to predict. If you're doing
 #' classification then this should be Y/N.
```

Comment:
I also realized that deploy only uses the predicted column name. Usually there are no values in that column. I changed it to say that and added to df: #' @param df Dataframe whose columns are used for new predictions. Data structure should match development as 
#' much as possible. Number of columns, names, types, grain, and predicted must be the same.